### ISSUE TITLE
Setting xmlDeclaration and doctypeDeclaration to false generates a "falsefalse" preamble

### ISSUE THREAD

**Issue by jraoult** [jkphl/svg-sprite#89]
Reproduced in version 1.2.2

I configure `svg-sprite` like that:
```javascript
{
      svg: {
        xmlDeclaration:  false
        doctypeDeclaration: false
      },
      transform: [{
        svgo: {
          plugins: [{
            removeUnknownsAndDefaults: false
          }]
        }
      }],
      mode: {
        symbol: {
          dest: '.',
          sprite: 'sprite.svg'
        }
      }
    }
```

I get an output file that starts with `falsefalse<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><symbol...`. Seems like the boolean value is treated as a string instead of disabling the declarations.

**Comment by jraoult** [comment]
Note: a workaround is to set the value of both properties to a blank space (empty string won't work)

**Comment by jkphl** [comment]
Hi @jraoult,

thanks for reporting this. Interesting. I'll look into this — it definitely worked at some point. Could you please tell me a little more about your platform (OS, Node version etc.). Thanks!

Cheers,
Joschi

**Comment by jraoult** [comment]
@jkphl yes sure. I'm on a mac os (Yosemite). I reproduced on Node 0.12.2 and iojs 2.2.1. Last time I had it working was the 1.1.0 of `svg-sprite`.

My guess is that this bug was introduced when the `rootAttributes ` setting was added.

**Comment by jkphl** [comment]
Should be fixed as of [release v1.2.3](https://github.com/jkphl/svg-sprite/releases/tag/v1.2.3).
